test/modules/logicModule/oracle/OptimisticOracleIntegrator.t.sol

```diff
@@ -202,7 +202,7 @@ contract OptimisticOracleIntegratorTest is ModuleTest {
         ooIntegrator.setOptimisticOracle(address(0));
     }
 
-    //@todo this Test is not correctly set @0xNuggan needs to be fixed
+    //::@todo this Test is not correctly set @0xNuggan needs to be fixed
     /*  function testSetOptimisticOracleFails_WhenNewOracleIsNotUmaOptimisticOracle(
```

Comment:
Sadly oO_v3 doesn't implement erc165 so we don't really have a way to test it but try calling oov3.defaultIdentifier() and check if it reverts :/